eframe/src/native/run.rs

```diff
@@ -194,7 +202,12 @@ pub fn run_glow(
                 painter.destroy();
             }
             winit::event::Event::UserEvent(RequestRepaintEvent) => window.request_redraw(),
-            _ => (),
+            winit::event::Event::NewEvents(winit::event::StartCause::ResumeTimeReached {
```

Comment:
You should be able to combine these with `winit::event::Event::UserEvent(RequestRepaintEvent | winit::event::Event::NewEvents…`